tests/jquery.flot.navigate.Test.js

```diff
@@ -305,6 +305,60 @@ describe("flot navigate plugin", function () {
       });
     });
 
+    describe('touchZoom', function() {
+      it('should zoom the plot',function() {
+
+        plot = $.plot(placeholder, [
+            [
+                [0, 0],
+                [10, 10]
+            ]
+        ], options);
+
+        xaxis = plot.getXAxes()[0];
+        yaxis = plot.getYAxes()[0];
+
+        var coords = new Array();
+        coords[0] = {
+            x : plot.getXAxes()[0].p2c(2) + plot.offset().left,
+            y : plot.getYAxes()[0].p2c(2) + plot.offset().top
+        }
+        coords[1] = {
+            x: plot.getXAxes()[0].p2c(6) + plot.offset().left,
+            y: plot.getYAxes()[0].p2c(6) + plot.offset().top
+        }
+
+        simulate.sendTouchEvents(coords, placeholder[0].childNodes[2], "touchstart");
```

Comment:
placeholder[0].childNodes[2] seems a little arbitrary. Give it a name, like

var eventHolder = placeholder.find('.flot-overlay')[0];